frontend/src/components/layout/AppNavBar/AppNavBar.test.tsx

```diff
@@ -225,117 +207,139 @@ describe('AppNavBar', () => {
       });
     });
 
-    it('AppNavBar include Disposal Project Link', () => {
-      (useKeycloak as jest.Mock).mockReturnValue({
-        keycloak: {
-          subject: 'test',
-          userInfo: {
-            roles: ['admin-properties', 'admin-projects'],
+    describe('AppNavBar Disposal Projects dropdown', () => {
+      it('AppNavBar include Disposal Projects dropdown', () => {
+        (useKeycloak as jest.Mock).mockReturnValue({
+          keycloak: {
+            subject: 'test',
+            userInfo: {
+              roles: [Claims.ADMIN_PROPERTIES, Claims.ADMIN_PROJECTS],
+            },
           },
-        },
+        });
+        const { getByText } = render(
+          <Provider store={store}>
+            <Router history={history}>
+              <AppNavBar />
+            </Router>
+          </Provider>,
+        );
+        const element = getByText('Disposal Projects');
+
+        expect(element).toBeTruthy();
       });
+      it('AppNavBar include Create Disposal Project Link', () => {
+        (useKeycloak as jest.Mock).mockReturnValue({
+          keycloak: {
+            subject: 'test',
+            userInfo: {
+              roles: [Claims.ADMIN_PROPERTIES, Claims.ADMIN_PROJECTS],
+            },
+          },
+        });
+        const { getByText } = render(
+          <Provider store={store}>
+            <Router history={history}>
+              <AppNavBar />
+            </Router>
+          </Provider>,
+        );
+        fireEvent.click(getByText('Disposal Projects'));
+        const link = getByText('Create Disposal Project');
 
-      const { getByText } = render(
-        <Provider store={store}>
-          <Router history={history}>
-            <AppNavBar />
-          </Router>
-        </Provider>,
-      );
-      const link = getByText('Dispose Properties');
-      expect(link).toBeTruthy();
-    });
-  });
+        expect(link).toBeTruthy();
```

Comment:
Forgot to mention this in our call, `toBeTruthy()` I believe is best used with `queryByText` and `toBeInTheDocument()` is best used with `getByText`. So in this case me may want to have `expect(link).toBeInTheDocument()`

Reply:
I know this wasn't you're doing though, was like this before. Maybe even by me earlier on 😆

Reply:
should I change it?

Reply:
yes, I'd rather toBeInTheDocument and toBeVisible. To be truthy should only be used for boolean evaluation, not for whether or not an element is visible